controller/spawn/kubernetes.go

```diff
@@ -31,6 +31,12 @@ func (s *KubernetesSpawner) Spawn(d *Daemon) error {
 	}
 	client := action.NewInstall(actionConfig)
 	client.ChartPathOptions.RepoURL = "https://filecoin-project.github.io/helm-charts"
+	if d.HelmChartRepoUrl != "" {
+		client.ChartPathOptions.RepoURL = d.HelmChartRepoUrl
+	}
+	if d.HelmChartVersion != "" {
+		client.ChartPathOptions.Version = d.HelmChartVersion
+	}
 	chartPath, err := client.ChartPathOptions.LocateChart("lotus-bundle",
```

Comment:
I wonder if the actual chart should be configurable also? -- although there is no garuantee that it would work of course unless it somehow had the same values file.

I'm thinking if we end up making some snowflake bots that uses a different chart, it might have the same url, but a different chart name. That hopefully never happens, but that seems as likely as having the same chart name with a different repo url. I don't have a strong opinion about this, just thinking if one is configurable the other may as well be.